justfile

```diff
@@ -21,16 +23,22 @@ linter-setup:
 create-vanilla-deployment-file:
    kustomize build {{kustomizeVanillaOverlay}} --output deploy/k8s/cloudbeat-ds.yaml
 
-build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image deploy-cloudbeat
+build-deploy-cloudbeat $GOOS=LOCAL_GOOS $GOARCH=LOCAL_GOARCH: load-cloudbeat-image
+  build-cloudbeat $GOOS $GOARCH
+  deploy-cloudbeat
```

Comment:
@oren-zohar load-cloudbeat-image should be the last command isn't it?

Reply:
nice catch, thanks!